### PR TITLE
Changing implementation From CollectionView to TableView

### LINKED ISSUE #352: Changing Implementation from CollectionView to TableView
**Actual Behaviour**

Currently CollectionView is implemented . It has to be changed to TableView

**Expected Behaviour**

When CollectionView is converted to TableView

**Steps to reproduce it**

N/A

**LogCat for the issue**

N/A
**Screenshots of the issue**

N/A

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by SIMRAN1**
Fixes #352 

Changes: Collection View is  being Changed to TableView.

Screenshots for the change: 
N/A

### CODE REVIEW

**Comment by SIMRAN1** [comment]
@imjog I need your help. I am stuck at this . I have changed the implementation to Table View but problem is coming that when I click the textField a black area is added at bottom. Please help me in resolving it.

**Comment by imjog** [comment]
@SIMRAN1 You have changed `collectionView` to `tableView` but didn't changes `collectionViewCells` to `tableViewCells` thats why it is showing black area.

File: Podfile.lock [hunk 1]
```diff
@@ -114,4 +114,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 685a75a46dea55bf0e30c875c369376f08cbf355
 
-COCOAPODS: 1.5.2
+COCOAPODS: 1.5.3
```
Comment (imjog) [inline Podfile.lock]: Please do not do this. You should have update cocoapod version in separate PR.

File: Susi/Controllers/ChatViewController/CollectionViewDelegate.swift [hunk 2]
```diff
@@ -8,82 +8,82 @@
...
+        tableView?.separatorColor = .clear
+        tableView?.delegate = self
+        tableView?.frame = CGRect(x: 0, y: 20, width: view.frame.width, height: view.frame.height - 68)
+        tableView?.register(IncomingBubbleCell.self, forCellReuseIdentifier: ControllerConstants.incomingCell)
+        tableView?.register(OutgoingChatCell.self, forCellReuseIdentifier: ControllerConstants.outgoingCell)
+        tableView?.register(RSSCell.self, forCellReuseIdentifier: ControllerConstants.rssCell)
+        tableView?.register(ActivityIndicatorCell.self, forCellReuseIdentifier: ControllerConstants.indicatorCell)
+        tableView?.register(MapCell.self, forCellReuseIdentifier: ControllerConstants.mapCell)
+        tableView?.register(AnchorCell.self, forCellReuseIdentifier: ControllerConstants.anchorCell)
+        tableView?.register(StopCell.self, forCellReuseIdentifier: ControllerConstants.stopCell)
```
Comment (imjog) [inline Susi/Controllers/ChatViewController/CollectionViewDelegate.swift]: cells are still collectionViewCell

**Comment by SIMRAN1** [comment]
@imjog I have changed to TableView cell

**Comment by imjog** [comment]
@SIMRAN1 Sorry, I stuck with other issues. Looking into this soon.

**Comment by imjog** [comment]
@SIMRAN1 close it for now, have a lot of bugs

**Comment by SIMRAN1** [comment]
okk @imjog !!

**Pull request closed without merging**